### DISCUSSION TITLE
Firefly to bitpanda Wallet

### DISCUSSION THREAD

**Discussion by PascalvT** [iotaledger/firefly#2660] [Help & Support] [answered]
Hello, I have a big problem :(
I did send my IOTA from Firefly Wallet to my bitpanda Wallet.
I felt 3 small transaction. These worked! The last transaction did not work and that was the largest position.
The transaction was confirmed by the tangle, yet she has not arrived. The transaction was conducted against about 10 hours ago.

please help me

**Comment by MiKeF05** [comment, marked as answer]
If the transaction is confirmed on the tangle you need to ask the support of Bitpanda.

**Reply by PascalvT** [reply]
Confirmed by milestone [2880219]

So if the transaction is Confirmed, are IOTA safe too?

thx

**Comment by DerSauso** [comment]
I have a similar problem as posted above. Did you already receive your funds? Do you already have an answer of the Bitpanda support team?

**Reply by PascalvT** [reply]
Hi, ich habe mit dem Support gesprochen und die Transaktion wurde jetzt gerade in Bitpanda angezeigt. Auf die frage "woran es gelegen hat" habe ich noch keine Rückmeldung erhalten.
So hat die Transaktion knapp 16 Stunden anstrengende gedauert. :O

Hi, I spoke to support and the transaction has just appeared in Bitpanda. I haven't received any response to the question "what was the reason?"
So the transaction took almost 16 exhausting hours. :O

**Reply by MiKeF05** [reply]
Leider ist das bei zentralen Exchanges sehr oft der Fall, dass das Verbuchen der Transaktion länger dauert.
